bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php

```diff
@@ -259,6 +260,9 @@ public function getCustomLayoutAction(Request $request)
                     );
 
                     $customLayout->setId($request->get('id'));
+                    if (!$customLayout->isWriteable()) {
+                        throw new ConfigWriteException();
```

Comment:
phpdocblock should have `* @throws ConfigWriteException|DataObject\Exception\DefinitionWriteException`